static/app/components/events/interfaces/frame/stacktraceLink.tsx

```diff
@@ -170,7 +181,11 @@ export function StacktraceLink({frame, event, line}: StacktraceLinkProps) {
 
   // Hide stacktrace link errors if the stacktrace might be minified javascript
   // Check if the line starts and ends with {snip}
-  const hideErrors = event.platform === 'javascript' && /(\{snip\}).*\1/.test(line);
```

Comment:
Do you have any idea what is this reference to snip?

Reply:
the backend does adds {snip} when the source line is too long, usually it means the code is minified without source maps. We're still attempting to show stacktrace links, but this will hide the "Tell us where your code is" if it fails